### ISSUE TITLE
User Log Colors Issue

### ISSUE THREAD

**Issue by Tangol** [villagedefrance/OpenCart-Overclocked#133]
According to the User Log Help, we have:
`Warning messages will be highlighted in RED. These include: Cleaning log, Authorization process fails, Attempt to access a restricted page.`

So from my understanding, if a user that doesn't have enough permissions, tries to access a restricted page, the log entry would be marked with red (**Allowed 0**).

![log](https://user-images.githubusercontent.com/3336728/36356437-c3ca52b2-14fa-11e8-840d-f3aa2df0f835.jpg)

The same applies for the login:
`A successful Login will be highlighted in GREEN.`
As you can see, the login was unsuccessful (**Allowed 0**), but still marked with green.

So basically **Allowed 0** means **RED**.

This is very important, as it helps us pinpoint suspicious users and Brute Force attacks.
